internal/admission/admission.go

```diff
@@ -31,12 +35,14 @@ func WithAdmissionHandlers(handlers ...AdmissionHandler) OptionsFunc {
 }
 
 type Router struct {
-	handlers map[string]AdmissionHandler
+	handlers    map[string]AdmissionHandler
+	limitRanger LimitRanger
 }
 
-func NewRouter(opts ...OptionsFunc) (*Router, error) {
+func NewRouter(lr LimitRanger, opts ...OptionsFunc) (*Router, error) {
```

Comment:
The signature change is causing an error in the `cli` package.

```
admissionRouter, err := admission.NewRouter(lr)
```

Reply:
fixed.